core/app/models/spree/adjustment.rb

```diff
@@ -73,10 +73,6 @@ class Adjustment < Spree::Base
     extend DisplayMoney
     money_methods :amount
 
-    def closed?
```

Comment:
@siddharth28 - Or if there is something I am missing, can you provide a description?